frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/shortcuts/command_shortcuts.dart

```diff
@@ -68,3 +70,14 @@ final _codeBlockLocalization = CodeBlockLocalizations(
 final localizedCodeBlockCommands = codeBlockCommands(
   localizations: _codeBlockLocalization,
 );
+
+final CommandShortcutEvent customCopyToClipboardCommand = CommandShortcutEvent(
+  key: 'copy to clipboard',
+  getDescription: () => 'Copy to clipboard',
+  command: 'ctrl+shift+c',
+  handler: (editorState) {
+    // Implement the handler for the 'Copy to clipboard' functionality
```

Comment:
The definition of `customCopyToClipboardCommand` makes no sense here. It's meant for documents, but I noticed you use it in the share menu.